src/Pages/EventsPage.js

```diff
@@ -11,6 +11,10 @@ export default function Events() {
   // called when DOM (page) is loaded
   useEffect(() => {
     setLoading(true);
+
+    const selectedYear = localStorage.getItem("year");
+    const selectedClass = localStorage.getItem("class");
+
```

Comment:
Is it possible to get the information on the year of study and class from just the single entry of a class code, eg. scci/2017, rather than the user having to enter Year 3 and then B. Tech CT?

Reply:
@leehonel sorry, I had not seen this. But yes, this is what we implemented in the sort of final build.